memlab.bib

```diff
@@ -45021,67 +44853,19 @@ @article{StadEtal99
-1.	Kep track and letter mem have lowest reliabilities of all tasks in study (.65 {\&}amp; .51)
-
-*	Shifting (switch trials RT -- no-switch trials RT DV):
-1.	Number letter (classify numbers or letters)
-2.	Color-shape (classify color or shape)
-3.	Category-switch task (animacy classification or size of word classification)
-
-Intelegence (Number correct DV -- except IQ)
-*	Gf
-1.	Ravens
-2.	WAIS Block Design
-*	Gc
-1.	Vocab test (synonyms or defns)
-2.	WAIS Information
-*	WAIS IQ as composite of Gf {\&}amp; Gc
-
-Results:
-
-A CFA model based on the 3 EF LVs and the 2 intelegence LVs had good fit
-
-Pattern of correlations between EF composits agreed with Miyake et al.'s 2000 study (correlated but separate EFs)
-
-
-Only updating was sig correlated with Gf (constraining all correlations to be equal worsened fit) (updating = .64, inhibiting = .29, shifting = .13)
-
-All EFs were correlated with Gc but updating was higher than inhibiting or shifting (.69 vs .31 {\&}amp; .31) and again constraining them to be equal worsened fit
-
-In an SEM specifying paths between the 3 EFs and both Gf and Gc, only updating had significant paths to either Gf or Gc. Suggests that the relationship between Gc and shifting and inhibiting in the CFA was largely due to their correlation with updating. Also, the correlation between Gf and Gc was reduced (.62 to .17) in the SEM indicating that a large part of the original correlation was due to the influence of updating (and the other EFs) on both.
-
-Discussion
-
-Seem to suggest toward the end the current intelegence tests fail to tap inhibiting and shifting rather than that inhibiting and shifting are not important in intelegence -- but no strong position is taken
-
-ME: Do the intelegence measures they chose preclude correlations with inhibiting and shifting?
-
-E.g., for Gc they used a vocab test and a general knowledge test -- things known to be robust to deficiencies in inhibition. It is less clear for the Gf measures but both are visual reasoning tasks{\ldots} It is conceivable that they are inherently sensitive to upating (e.g., generating and holding hypotheses) and less sensitive to the other functions (there is not a lot of stimuli involved in a trial and the constant perceptual display of the patterns in ravens could make inhibiting past trials less of an issue). But it could be argued that the same results were found with the global IQ score which should be less selective.
-},
-	Author = {Friedman, NP and Miyake, A and Corley, RP and Young, SE and DeFries, JC and Hewitt, JK},
-	Date-Added = {2014-04-04 16:37:02 +0000},
-	Date-Modified = {2014-04-04 16:37:21 +0000},
-	Journal = {Psychological Science},
-	Month = jan,
-	Number = {2},
-	Pages = {172--179},
-	Title = {Not all executive functions are related to intelligence},
-	Volume = {17},
-	Year = {2006}}
+@Article{FrieEtal06,
+  author        = {Friedman, NP and Miyake, A and Corley, RP and Young, SE and DeFries, JC and Hewitt, JK},
```

Comment:
NP Friedman and A Miyake and RP Corley and SE Young and JC De{F}ries and JK Hewitt